app/controllers/aid_applications/disbursements_controller.rb

```diff
@@ -61,10 +65,15 @@ def aid_application_params
     class SearchCard
       include ActiveModel::Model
       attr_accessor :sequence_number
+      attr_accessor :sequence_number_confirmation
 
       def self.model_name
         ActiveModel::Name.new(self, nil, "SearchCard")
       end
+
+      def matching_sequence_numbers?
+        return true unless sequence_number_confirmation.blank? || sequence_number_confirmation != sequence_number
```

Comment:
Anytime you have a `return boolean if...` you should consider rewriting it solely as the boolean statement. `!(sequence_number_confirmation.blank? || sequence_number_confirmation != sequence_number)`

...which is a bit nasty with the double negatives.

`sequence_number_confirmation.present? && sequence_number_confirmation == sequence_number`

That's style, I think there is a slight logic change though that's needed because this check is skipped entirely if the `sequence_number_confirmation` is blank.

I think it should be more like: 

`(sequence_number.presence || '').strip == (sequence_number_confirmation.presence || '').strip`

Reply:
Thanks! I'll incorporate this.